README.md

```diff
@@ -272,7 +272,23 @@ should mention how to instantiate those variables/files, etc.
 
 ### Testing
 
-(TODO: Write me)
+We believe testing is an important part of producing high quality
+software. Specifically, we believe robust testing helps improve
+maintaiability as well as reliability of the produced software.
+
+Some general recommendations are:
+* Spend time implement testing.
+* Use established testing and mocking framework for the project 
+  language/environment (e.g. JUnit, pytest, the Go testing package).
```

Comment:
maybe change to "Use established testing framework (e.g. JUnit, pytest and Go testing package) for your project.". 

"Testing and mocking framework" sounds redundant.

Reply:
One might think :)

It may be an anomaly, but I feel it's been rather that people pick the platform "standard" testing framework, but mocking has been much less stringent, meaning different projects have picked different frameworks. The fact that they tend to use very similar naming schemes doesn't always make using them easier.